provider-tutorials/provider-cli.md

```diff
@@ -229,3 +231,138 @@ Run `ya-provider keystore --help` to see possible subcommands
 The [Computational Payload Manifests](../requestor-tutorials/vm-runtime/computation-payload-manifest.md) embedded in the Demands can specify a list of URLs which may get called by the services running on a Provider. If the manifest declares requests to URLs in domains that are not whitelisted, it must come with a [signature and app author's public certificate](../requestor-tutorials/vm-runtime/computation-payload-manifest.md). By default, the domains `whitelist` consists of a curated set of public websites and APIs like github, dockerhub or public Ethereum nodes.
 
 Run `ya-provider whitelist --help` to see possible subcommands
+
+### Rules
+
+`ya-provider` offers a mechanism that allows to control the Rules for specific features i.e. Outbound Network.
+
+#### Outbound Network feature
```

Comment:
```suggestion
#### Outbound Network feature
```

Why do we need to attach a title of "feature" to a feature? it's like saying "Master and Margaret book" instead of just "Master and Margaret" ...